### ISSUE TITLE
Error in microwin

### ISSUE THREAD

**Issue by Papoutsisss** [ChrisTitusTech/winutil#2097]
Selected value '6 : Windows 11 Pro'.....

Could not remove OS package Package_for_KB502739731bf3856ad364e35amd6422621.2355.1.1
Could not remove OS package Package_for_RollupFix31bf3856ad364e35amd6422621.2861.1.6
Invoke-WPFMicrowin : An unexpected error occurred: Removal failed. Please contact your software vendor.
At line:2805 char:24

    "WPFMicrowin" {Invoke-WPFMicrowin}
                   ~~~~~~~~~~~~~~~~~~
CategoryInfo : NotSpecified: (:) [Write-Error], WriteErrorException
FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,Invoke-WPFMicrowin
Unmounting Registry...
ERROR: The parameter is incorrect.
ERROR: The parameter is incorrect.
ERROR: The parameter is incorrect.
ERROR: The parameter is incorrect.
ERROR: The parameter is incorrect.
Cleaning up image...
Cleanup complete.

**Comment by blusewill** [comment]
Try using Powershell 7 to see if it works or not.
Sometimes Powershell 5 is buggy sometimes

**Comment by CodingWonders** [comment]
It should be fixed with #2192

**Comment by Real-MullaC** [comment]
@Papoutsisss if this issue is fixed please could you close this issue?
